docs/source/tutorial/modules.rst

```diff
@@ -301,15 +301,15 @@ Look at the following example.
     useNSFromOutside' : String
     useNSFromOutside' = Y.g "x" -- value is "xa"
 
-Parameterised blocks
+Parameterised blocks - `parameters`-blocks
 ====================
```

Comment:
`rst` requires the underline to be extended too

Reply:
should be fixed